docs/contracts.rst

```diff
@@ -35,4 +35,6 @@ There is no "cron" concept in Ethereum to call a function at a particular event
 
 .. include:: contracts/libraries.rst
 
-.. include:: contracts/using-for.rst
\ No newline at end of file
+.. include:: contracts/using-for.rst
+
+.. include:: contracts/storage-layout-specifier.rst
```

Comment:
IMO it would fit better somewhere closer to storage variables. E.g. before the page about functions.